libs/wire-api/src/Wire/API/Routes/Public/Galley.hs

```diff
@@ -1178,6 +1185,48 @@ type AllFeatureConfigsGet =
         :> Get '[Servant.JSON] AllFeatureConfigs
     )
 
+type AllFeaturesGet =
+  Named
+    "get-all-features"
+    ( Summary "Shows the configuration status of every team feature"
+        :> CanThrow 'NotATeamMember
+        :> CanThrow OperationDenied
+        :> CanThrow 'TeamNotFound
+        :> ZUser
```

Comment:
I'd suggest to make this `ZLocalUser` instead of `ZUser`. That way we provide more information to the endpoint and have a local value in scope should it be needed.